featurebyte/service/feature.py

```diff
@@ -198,3 +200,23 @@ async def update_readiness(self, document_id: ObjectId, readiness: FeatureReadin
             user_id=self.user.id,
             disable_audit=self.should_disable_audit,
         )
+
+    async def update_last_updated_date(
+        self, aggregation_id: str, last_updated_by_scheduled_task_at: datetime
+    ) -> None:
```

Comment:
Would be good to make the method name clearer.

Reply:
will change the method name